src/aria2p/client.py

```diff
@@ -201,14 +203,27 @@ def __init__(
         self.port = port
         self.secret = secret
         self.timeout = timeout
+        self.retries = retries
         self.listening = False
+        self.__httpx_session = None
 
     def __str__(self):
         return self.server
 
     def __repr__(self):
         return f"Client(host='{self.host}', port={self.port}, secret='********')"
 
+    def __del__(self):
+        if self.__httpx_session is not None:
+            self.__httpx_session.close()
+
+    @property
+    def _session(self):
+        if self.__httpx_session is not None:
+            return self.__httpx_session
+        self.__httpx_session = httpx.Client(timeout=self.timeout, transport=httpx.HTTPTransport(verify=False, retries=self.retries))
```

Comment:
Why `verify=False`? Why retries? It looks like `requests` does not add retries automatically. By the way, 10 would be too much IMO.